src/utils/index.ts

```diff
@@ -0,0 +1,16 @@
+import * as vscode from "vscode";
+
+export async function validateAndRecommendExtension(extName: string, message: string): Promise<boolean> {
+  const ext = vscode.extensions.getExtension(extName);
+  if (ext) {
+    return true;
+  }
+
+  const action = "Details";
```

Comment:
Should add an `Install` button in the recommend dialog.

Reply:
There is no API in vscode which allows you to "Install" an extension directly. So the workaround is to show the "Details" page and then you can click the "Install" button to achieve that.

Reply:
VS Code's recommendation has the `Install` button, could we ask for their implementation?